apps/user-portal/src/components/shared/header.tsx

```diff
@@ -204,7 +204,12 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
                             ? state.productName
                             : config.ui.productName
                     }
-                    version={ config.deployment.productVersion }
+                    version={ config.ui.productVersionConfig?.override ?? config.deployment.productVersion }
```

Comment:
Can we keep the product version key in UI config object config.ui.productVersionConfig.productVersion instead of config.ui.productVersionConfig.override?

Reply:
`config.ui.productVersionConfig?.override` this is for overriding the `config.deployment.productVersion`. If `override` is not intuitive enough we can go with `productVersionOverride`. WDYT?

Reply:
Renamed `override` -> `versionOverride` in `#87780dc`